x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -0,0 +1,193 @@
+import { i18n } from '@kbn/i18n';
+import {
+  MlInMemoryTable,
+  SortDirection,
+  SORT_DIRECTION,
+  OnTableChangeArg,
+} from '../../../components/ml_in_memory_table';
+import { formatHumanReadableDateTimeSeconds } from '../../../util/date_utils';
+import { ExplorerLink } from './actions';
+import { getJobsFromGroup } from './utils';
+import { Groups, Group, Jobs } from './anomaly_detection_panel';
+// @ts-ignore
+import { StatsBar, JobStatsBarStats } from '../../../components/stats_bar';
+// @ts-ignore
+import { JobSelectorBadge } from '../../../components/job_selector/job_selector_badge';
+// @ts-ignore
+import { toLocaleString } from '../../../util/string_utils';
+import { getSeverityColor } from '../../../../common/util/anomaly_utils';
+
+// Used to pass on attribute names to table columns
+export enum AnomalyDetectionListColumns {
+  id = 'id',
+  maxAnomalyScore = 'max_anomaly_score',
+  jobIds = 'jobIds',
+  latestTimestamp = 'latest_timestamp',
+  docsProcessed = 'docs_processed',
+}
+
+interface Props {
+  items: Groups;
+  statsBarData: JobStatsBarStats;
+  jobsList: Jobs;
+}
+
+export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData }) => {
+  const groupsList = Object.values(items);
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(10);
+
+  const [sortField, setSortField] = useState<string>(AnomalyDetectionListColumns.id);
+  const [sortDirection, setSortDirection] = useState<SortDirection>(SORT_DIRECTION.ASC);
+
+  // columns: group, max anomaly, jobs in group, latest timestamp, docs processed, action to explorer
+  const columns: any[] = [
+    {
+      field: AnomalyDetectionListColumns.id,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableId', {
+        defaultMessage: 'Group ID',
+      }),
+      render: (id: Group['id']) => <JobSelectorBadge id={id} isGroup={id !== 'ungrouped'} />,
+      sortable: true,
+      truncateText: true,
+      width: '20%',
+    },
+    {
+      field: AnomalyDetectionListColumns.maxAnomalyScore,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScore', {
+        defaultMessage: 'Max anomaly score',
+      }),
+      sortable: true,
+      render: (score: Group['max_anomaly_score']) => {
+        if (score === null) {
+          return <EuiLoadingSpinner />;
+        } else {
+          const color: string = getSeverityColor(score);
+          return (
+            // @ts-ignore
+            <EuiHealth color={color} compressed="true">
+              {score >= 1 ? Math.floor(score) : '< 1'}
+            </EuiHealth>
+          );
+        }
+      },
+      truncateText: true,
+      width: '150px',
+    },
+    {
+      field: AnomalyDetectionListColumns.jobIds,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableNumJobs', {
+        defaultMessage: 'Jobs in group',
+      }),
+      render: (jobIds: Group['jobIds']) => jobIds.length,
+      sortable: true,
```

Comment:
I am seeing some odd sort behavior here. It doesn't seem to sort numerically:

![image](https://user-images.githubusercontent.com/7405507/65152271-89aa6e00-da1f-11e9-9c7b-ed4eed767d18.png)

Reply:
Added to follow-up meta issue 👍